site/docs/concepts/optimizer.md

```diff
@@ -0,0 +1,26 @@
+# Optimizer
+
+The optimizer component builds an optimal data-plane plotter for a given `FybrikApplication` custom resource, taking into account:
+* Available resources (modules, clusters, storage accounts)
+* Application specification (e.g., geography, data access protocol)
+* Specifications of the data-sets required by the application
+* Governance actions required by the data-governance policy manager
+* [Configuration policies](./config-policies)
+* Optimization goals
```

Comment:
Optimization goals are part of the configuration policies.  Need to mention hard and soft it config policies